mne/decoding/time_gen.py

```diff
@@ -1206,11 +1248,21 @@ class TimeDecoding(_GeneralizationAcrossTime):
                 each of the k-fold cross-validation classifiers, and average
                 these predictions into a single estimate per sample.
 
-        Default: 'cross-validation'
+        Defaults to 'cross-validation'.
     scorer : object | None | str
-        scikit-learn Scorer instance or str type indicating the
-        name of the scorer such as `accuracy`, `roc_auc`. If None,
-        set to accuracy_score.
+        scikit-learn Scorer instance or str type indicating the name of the
+        scorer such as `accuracy`, `roc_auc`. If None, set to accuracy_score.
+    score_mode : {'per-fold', 'mean-across-folds', 'no-cv'}
+        Determines how the scorer is estimated:
+
+            ``per-fold`` : returns the score obtained in each fold.
```

Comment:
(or actually I suspect just an extra newline between the single-line ones would work -- you can try it locally with `make html_dev-noplot` within the `doc` directory if you have all the doc-building requirements up to date)